pkg/event/event_marshal_test.go

```diff
@@ -31,14 +32,14 @@ func TestMarshal(t *testing.T) {
 	now := types.Timestamp{Time: time.Now().UTC()}
 	schemaURL := "http://example.com/schema"
 	resource := "/cluster/node/ptp"
-	_type := "ptp_status_type"
+	_type := ptp.STATUS
```

Comment:
same here, EventPtpStateChange and remove const ptp.STATUS

Reply:
done